packages/arb-token-bridge-ui/src/components/common/Header.tsx

```diff
@@ -37,43 +31,9 @@ export function Header({ children }: { children?: React.ReactNode }) {
           alt="Arbitrum"
         />
         {isTestnet && <span className="grow font-medium">TESTNET MODE</span>}
-        <div className="hidden sm:flex">{children}</div>
+        <div className="absolute top-4 hidden sm:flex">{children}</div>
```

Comment:
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/f9f03c46-bba4-4b28-aa12-c2028aefbfb4)
this div is off centre vertically
look at the Account element